### DISCUSSION TITLE
DMs not working

### DISCUSSION THREAD

**Discussion by sobraniechrome** [drizzle-mizzle/CharacterAI-Discord-Bot#53] [Q&A] [answered]
I'm probably just dumb but I can't get the DMs to work; when I try to DM the bot, it gives the Clyde error message that it can't receive anything.  
![image](https://user-images.githubusercontent.com/130184017/230674023-cdeadc56-1a9f-49ca-bfb7-946e42472c8b.png)

How do I fix this? Again sorry this is stupid.

**Comment by drizzle-mizzle** [comment, marked as answer]
Do you actually share server with your bot?
Do you have DMs enabled on your server?

Right click your server

![image](https://user-images.githubusercontent.com/55811932/230689643-d247ac91-e8ea-4480-842a-3dbe1832636a.png)

Go to "privacy settings"

![image](https://user-images.githubusercontent.com/55811932/230689742-911d9cea-5adf-459b-a66c-a4ac589f7671.png)

Window similar to this will appear. Switch Allow DMs to enabled.

**Reply by sobraniechrome** [reply]
That worked! Thank you so much and sorry again! <3
